packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

```diff
@@ -385,7 +385,7 @@ export const StaticProperties: ObjectMap2<CoreJSPolyfillDescriptor> = {
       "es.object.create",
       "es.object.freeze",
     ]),
-    stringify: define("json/stringify", ["es.json.stringify"], "es.symbol"),
+    stringify: define("json/stringify", ["es.json.stringify"]),
```

Comment:
It's possible that the reason there was `es.symbol` is that when polyfilling `JSON.stringify` we also have to polyfill symbols. In that case, this definition should be
```suggestion
    stringify: define("json/stringify", ["es.json.stringify", "es.symbol"]),
```

Let's wait for @zloirock to see this PR before merging :)

Reply:
Thanks, fixed.